### PR TITLE
NAV-185-disabled-action-button-tooltips

### PR DESCRIPTION
**Pull request by Manoj-nathwani**
- https://fjelltopp.atlassian.net/browse/NAV-185

### CODE REVIEW

**Comment by Manoj-nathwani** [comment]
@jonathansberry: Do we ever want to disable the `What's Next?` button if `terminus: true`.
I think we should.

**Comment by jonathansberry** [comment]
> @jonathansberry: Do we ever want to disable the `What's Next?` button if `terminus: true`. I think we should.

No we don't.  They may have changed something in the underlying data that changes the result of the what's next button.

File: lib/actionButtons.js [hunk 1]
```diff
@@ -109,19 +121,28 @@ export function priorTaskButton(workflowState) {
...
+        enabled: false,
+        tooltip: ['This workflow is now complete']
+      };
     } else if (isLatestTask) {
-      return false;
+      return {
+        enabled: false,
+        tooltip: [
+          'This is your current task.',
+          'Click "What\'s Next ? " to proceed.'
```
Comment (jonathansberry) [inline lib/actionButtons.js:141]:
```suggestion
          'Click "What\'s Next?" to proceed.'
```

File: lib/actionButtons.js [hunk 2]
```diff
@@ -77,28 +89,28 @@ export function taskCompleteCheckbox(workflowState) {
...
+    enabled, tooltip, onClickAction,
+    checked: completed
   };
 };
 
 export function priorTaskButton(workflowState) {
   const { manual, isOldestTask } = getWorkflowStats(workflowState);
-  const enabled = (() => {
+  const { enabled, tooltip } = (() => {
+    const firstTaskTooltip = ['This is your first task'];
```
Comment (jonathansberry) [inline lib/actionButtons.js:102]:
```suggestion
    const firstTaskTooltip = ['This is your first task.'];
```

File: lib/actionButtons.js [hunk 3]
```diff
@@ -31,16 +31,28 @@ function errorCheckButton(workflowState, enabled, onClickAction) {
...
       if (manual) {
-        return true;
+        return { enabled: true };
       } else {
-        return false;
+        return {
+          enabled: false,
+          tooltip: [
+            'Task status is automatically checked by system.',
+            'Click "What\'s Next" once task is complete.',
```
Comment (jonathansberry) [inline lib/actionButtons.js:53]:
```suggestion
            'Click "What\'s Next?" once task is complete.',
```

File: lib/actionButtons.js [hunk 4]
```diff
@@ -109,19 +121,28 @@ export function priorTaskButton(workflowState) {
...
 
 export function nextTaskButton(workflowState) {
   const { isLatestTask, terminus } = getWorkflowStats(workflowState);
-  const enabled = (() => {
+  const { enabled, tooltip } = (() => {
     if (terminus) {
-      return false;
+      return {
+        enabled: false,
+        tooltip: ['This workflow is now complete']
```
Comment (jonathansberry) [inline lib/actionButtons.js:134]:
```suggestion
        tooltip: ['This workflow is now complete.']
```

**Review by jonathansberry — CHANGES_REQUESTED** [review]
Ok good. Thanks for the quick work.  I've proposed some typo fixing for the tooltip messages. Once those are addressed I am happy to merge. Make sure you take some time to manually test it, since there are no automated tests yet.

**Review by jonathansberry — APPROVED** [review]

**Pull request merged**